### ISSUE TITLE
DynamoDb endpoint

### ISSUE THREAD

**Issue by JonnyX11** [madeindra/signal-setup-guide#80]
Hey,

Can you please explain how you config in yml file the endpoint of DynamoDb of the docker?

Thanks!

**Comment by jacob-pro** [comment]
I don't think this can be done without modifying the source code of the Signal server...
It expects to be able to use the instance profile of the EC2 instance the server is running on to connect to a real AWS region

**Comment by JonnyX11** [comment]
So how did you make it works ?

What did you change to run the DynamoDB on docker ?

**Comment by jacob-pro** [comment]
I haven't made it work!
Using local DynamoDb will require some modifications to the code.
If this is something you want to do I suggest starting here:

https://github.com/signalapp/Signal-Server/blob/10cd60738ac086407f01d68700717b7e544739b0/service/src/main/java/org/whispersystems/textsecuregcm/WhisperServerService.java#L281

If you are not running in an Aws EC2 environment, that is not the only problem you will have, you will also have to look
at modifying the DynamicConfigurationManager, because the server will not start without access to Aws AppConfig:

https://github.com/signalapp/Signal-Server/blob/master/service/src/main/java/org/whispersystems/textsecuregcm/storage/DynamicConfigurationManager.java#L46

Like I said, I have not got this working myself, because after investigating it, I discovered it would require substantial code modifications. So I cannot offer any further advice. I am currently running Single Server v4.97 because that does not have the AWS dependencies.

**Comment by JonnyX11** [comment]
Do you have a conference call on this version Signal Server v4.97?

If yes can you share the steps please ?

**Comment by JonnyX11** [comment]
OK thanks

Please try to figure out if there is a conference call.

What about clients? which version android and iOS ?
